data/darktableconfig.xml.in

```diff
@@ -484,6 +505,132 @@
     <shortdescription>number of collections to be stored</shortdescription>
     <longdescription>the number of recent collections to store and show in this list</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/filtering/history_max</name>
+    <type min="1" max="50">int</type>
+    <default>10</default>
+    <shortdescription>number of filters to be stored</shortdescription>
+    <longdescription>the number of recent filters to store and show in history list</longdescription>
+  </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/filtering/sort_history_max</name>
+    <type min="1" max="50">int</type>
+    <default>10</default>
+    <shortdescription>number of sort orders to be stored</shortdescription>
+    <longdescription>the number of recent sort orders to store and show in history list</longdescription>
+  </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/filtering/num_rules</name>
+    <type>int</type>
+    <default>3</default>
+    <shortdescription>nb of rules by default</shortdescription>
+    <longdescription></longdescription>
+  </dtconfig>
+  <dtconfig>
```

Comment:
Not sure we want to create entries for cont based on a unbounded number (well bounded but large).

Reply:
My idea at some point was to add entries ending with * as a pattern and then remove all darktablerc entries not defined in this template file. This would be a nice way to clean-up old darktablerc entries. But for sure this in not for this PR.

Reply:
I'm not sure to understand : The entry here and below define the 3 filters that will be shown on first startup (rating, colorlabels and search)

Reply:
But user can add some more and then we will also create `plugins/lighttable/filtering/item3` and `plugins/lighttable/filtering/item4`... But I think I understand, those are just defining default values for the 3 filters that will be should at startup, that's it? In this case, it is ok.